config/initializers/dfe_analytics.rb

```diff
@@ -55,4 +55,6 @@
 
   # if part of any model name is migration, it wont be loaded
   config.excluded_models_proc = proc { |x| x.to_s =~ /Migration::/ }
+
+  config.entity_table_checks_enabled = true
 end
```

Comment:
could we add `config.user_identifier = proc { |user| user&.id if user.respond_to?(:id) }` here as well incase as we had issues in Ecf because of user?